packages/host/app/components/ai-assistant/code-block/patch-footer.gts

```diff
@@ -0,0 +1,33 @@
+import type { TemplateOnlyComponent } from '@ember/component/template-only';
+
+export interface CodeBlockPatchFooterSignature {
+  Args: {};
+  Element: HTMLElement;
+  Blocks: {
+    default: [];
+  };
+}
+
+const CodeBlockPatchFooterComponent: TemplateOnlyComponent<CodeBlockPatchFooterSignature> =
+  <template>
+    <style scoped>
+      .code-block-patch-footer {
+        display: flex;
+        align-items: center;
+        justify-content: space-between;
+        gap: var(--boxel-sp-xs);
+        background-color: var(--boxel-650);
+        color: var(--boxel-light);
+        padding: 8px 12px;
+        font-size: 14px;
+        height: 50px;
```

Comment:
This should probably be `min-height` to account for longer text.
<img width="1036" height="322" alt="long-text" src="https://github.com/user-attachments/assets/b1a27f04-79c3-4ca1-833f-24354d3715e3" />